go-controller/pkg/cni/cnishim.go

```diff
@@ -130,7 +116,13 @@ func setupLogging(conf *ovntypes.NetConf) {
 		if err := klogFlags.Set("alsologtostderr", "true"); err != nil {
 			klog.Warningf("Error setting klog alsologtostderr: %v", err)
 		}
-		klog.SetOutput(file)
+		klog.SetOutput(&lumberjack.Logger{
+			Filename:   conf.LogFile,
+			MaxSize:    100, // megabytes
+			MaxBackups: 5,
```

Comment:
Overall this patch looks good.
I'm curious to why the values are different from those in `./go-controller/pkg/config/config.go` and as to whether or not we out to expose them somewhere so they can be altered by users.

```
 klog.SetOutput(&lumberjack.Logger{
                        Filename:   Logging.File,
                        MaxSize:    100, // megabytes
                        MaxBackups: 10,
                        MaxAge:     30, // days
                        Compress:   true,
                })
```

Reply:
@dave-tucker I think it is a good idea to expose these as configuration option. In our clusters, we have a very small '/var/log' partition and it was getting filled up way too fast by OVN K8s log files. So, we changed it to smaller values at both the places internally.